android/capacitor.settings.gradle

```diff
@@ -10,3 +10,6 @@ project(':capacitor-keyboard').projectDir = new File('../node_modules/@capacitor
 
 include ':capacitor-status-bar'
 project(':capacitor-status-bar').projectDir = new File('../node_modules/@capacitor/status-bar/android')
+
+include ':webview-background'
+project(':webview-background').projectDir = new File('../node_modules/webview-background/android')
```

Comment:
Is there a reason the plugin is loaded from `node_modules` rather than directly from the source, since it's part of the same repo?

Reply:
So I added it like this so that it can flow through the same capacitor build flow as normal plugins. Its also easier to add changes to this custom plugin in the future if we need too. The code is already available here.

It first builds and then adds to the project through the normal flow after the plugin is added to the app by `yarn add ./em_plugins/webview`

Reply:
Okay, sounds good!